From 7496ad1a31f27442e2326f2c2a68c1c8b587db4c Mon Sep 17 00:00:00 2001
From: Aleksey <zaychukaleksey@gmail.com>
Date: Tue, 23 Jan 2024 16:20:43 +0300
Subject: [PATCH] Fix build with GCC 14.

The method should not be const since it mutates the member field, which is not marked as mutable.
---
 dtl/Diff.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dtl/Diff.hpp b/dtl/Diff.hpp
index 8c9c774..57067f9 100644
--- a/thirdparty/dtl/dtl/Diff.hpp
+++ b/thirdparty/dtl/dtl/Diff.hpp
@@ -164,7 +164,7 @@ namespace dtl {
             return trivial;
         }
         
-        void enableTrivial () const {
+        void enableTrivial () {
             this->trivial = true;
         }
         

